lfucg-exactions/server/accounts/viewsets.py

```diff
@@ -133,15 +117,20 @@ class PaymentViewSet(viewsets.ModelViewSet):
     serializer_class = PaymentSerializer
     queryset = Payment.objects.all()
     permission_classes = (CanAdminister,)
+    filter_backends = (DjangoFilterBackend, filters.SearchFilter,)
+    search_fields = ('lot_id__address_street', 'lot_id__address_number', 'payment_type', 'check_number', 'credit_account__account_name', 'paid_by', 'credit_source__resolution_number')
```

Comment:
With lot searches: you can search by address_full.  Otherwise, if you can search by the unique parts, you could eliminate address_full entirely, but will need to update data displays that reference address_full.

Reply:
IDK why I used street here... I have changed it to full. It accomplishes what we'd want it to do.